src/qt/paymentserver.cpp

```diff
@@ -356,24 +354,11 @@ void PaymentServer::handleURIOrFile(const QString& s)
 
     if (QFile::exists(s)) // payment request file
     {
-#ifdef ENABLE_BIP70
-        PaymentRequestPlus request;
-        SendCoinsRecipient recipient;
-        if (!readPaymentRequestFromFile(s, request))
-        {
-            Q_EMIT message(tr("Payment request file handling"),
-                tr("Payment request file cannot be read! This can be caused by an invalid payment request file."),
-                CClientUIInterface::ICON_WARNING);
-        }
-        else if (processPaymentRequest(request, recipient))
-            Q_EMIT receivedPaymentRequest(recipient);
-
-        return;
-#else
-        Q_EMIT message(tr("Payment request file handling"),
-            tr("Cannot process payment request because BIP70 support was not compiled in."),
-            CClientUIInterface::ICON_WARNING);
-#endif
+      Q_EMIT message(tr("Payment request file handling"),
+          tr("Cannot process payment request because BIP70 is not supported.")+
+          tr("Due to widespread security flaws in BIP70 it's strongly recommended that any merchant instructions to switch wallets be ignored.")+
```

Comment:
I think we probably want to remove this line in both errors, and maybe recommend using the Dash mobile wallets for Bip70 support?